flow/Net2.actor.cpp

```diff
@@ -1719,6 +1737,43 @@ INetwork* newNet2(const TLSConfig& tlsConfig, bool useThreadPool, bool useMetric
 	return N2::g_net2;
 }
 
+boost::system::error_code FailureInjector::rollRandomClose() const {
+	if (FLOW_KNOBS->ENABLE_CHAOS_FEATURES && FLOW_KNOBS->INJECT_CONNECTION_FAILURES &&
+	    deterministicRandom()->random01() < .000005) {
+		return boost::system::errc::make_error_code(boost::system::errc::network_reset);
+	}
+	return boost::system::error_code();
+}
+
+void FailureInjector::cloggFor(const Optional<NetworkAddress>& peer, double time) {
+	if (peer.present()) {
+		auto& until = clogConnection[peer.get()];
+		until = std::max(until, now() + time);
+	} else {
+		clogAllUntil = std::max(clogAllUntil, now() + time);
+	}
```

Comment:
It doesn't seem possible to reach this path right now, since this function is only called when a message is received for a specific peer. Should this be called even if the peer isn't present?

Reply:
In this call:
```
FailureInjector::injector()->clogFor(req.clog.get().address, req.clog.get().time);
```

`req.clog.get().address` is of type `Optional<Address>` (see `ClogCommand::address`). So I don't think what you'e saying is true? This code is reachable.